frontend/ui/lib/widgets/create_trip_widget.dart

```diff
@@ -0,0 +1,98 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:tripmeout/widgets/map_widget.dart';
+
+class CreateTripWidget extends StatefulWidget {
+  @override
+  _CreateTripWidgetState createState() => _CreateTripWidgetState();
+}
+
+class _CreateTripWidgetState extends State<CreateTripWidget> {
+  String place = "No Input";
+  int radius = 0;
+  String newInformation = 'Grabbed info placed here.';
+  changeText() {
+    setState(() {
+      newInformation = place + " " + radius.toString() + " km";
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Center(
+        child: FittedBox(
+      child: Column(
+        children: <Widget>[
+          Row(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: <Widget>[
+              Padding(
+                padding: const EdgeInsets.all(25.0),
+                child: Container(
+                  width: 250.0,
+                  child: TextFormField(
+                    onChanged: (text) {
+                      place = text;
+                    },
+                    textAlign: TextAlign.center,
+                    decoration: InputDecoration(
+                      border: OutlineInputBorder(),
+                      labelText: 'Enter your Location',
+                    ),
+                  ),
+                ),
+              ),
+              Padding(
+                padding: const EdgeInsets.all(25.0),
+                child: Container(
+                  width: 125.0,
+                  child: TextFormField(
+                    onChanged: (text) {
+                      radius = int.parse(text);
+                    },
+                    textAlign: TextAlign.center,
+                    keyboardType: TextInputType.number,
+                    inputFormatters: <TextInputFormatter>[
+                      WhitelistingTextInputFormatter.digitsOnly
+                    ],
+                    decoration: InputDecoration(
+                        border: OutlineInputBorder(), labelText: 'Radius KM'),
+                  ),
+                ),
+              ),
+            ],
+          ),
+          Row(
```

Comment:
nix:you really don't need a row widget if your only going to have one child and your not changing its alignment

Reply:
Wouldn't it affect whether the other things will be in the same row with it the widgets or not?

Reply:
I don't think so because everything is in a column widget

Reply:
I can try. But what I was saying is that if it's just one column I would think that it would just stretch as far is it could in the column. I put them in a row even if it's just one item for organizing and assuring it would be in the set place I wanted.